src/parse.c

```diff
@@ -4440,7 +4440,7 @@ void Parser::checkCstyleTypeSyntax(Loc loc, Type *t, int alt, Identifier *ident)
 
 Statement *Parser::parseStatement(int flags, const utf8_t** endPtr, Loc *pEndloc)
 {
-    Statement *s;
+    Statement *s = NULL;
```

Comment:
> I'm fine with you just setting it to NULL at the top, but the goto should work.

@WalterBright seems to dislike this, but I can do whatever you guys want. Adding the goto gets rid of the warning, and I will test with the update above. Like I said, I don't really feel comfortable modifying the code, but just doing the minimum possible to fix the warning :)